UnitTests/SqlPreparation/SqlPreparationQueryModelVisitorTest.cs

```diff
@@ -271,18 +272,17 @@ public void VisitMainFromClause_CreatesFromExpression ()
           preparedSqlTable, new Ordering[] { }, new SqlTableReferenceExpression (preparedSqlTable), null);
 
       _stageMock
-          .Expect (
+          .Setup (
               mock => mock.PrepareFromExpression (
-                  Arg<Expression>.Matches (e => e == _mainFromClause.FromExpression),
-                  Arg<ISqlPreparationContext>.Matches (c => c != _context),
-                  Arg<Func<ITableInfo, SqlTable>>.Is.Anything,
-                  Arg.Is (OrderingExtractionPolicy.ExtractOrderingsIntoProjection)))
-          .Return (preparedFromExpressionInfo);
-
-      _stageMock.Replay();
+                  It.Is<Expression> (e => e == _mainFromClause.FromExpression),
```

Comment:
Don't we need to use  ItExpr instead of It? (entire file, and search for It.Is<Expression>